lessons/specifics/mnesia.md

```diff
@@ -113,6 +113,13 @@ We define the columns using the atoms `:id`, `:name`, and `:job`. When we execut
  - `{:atomic, :ok}` if the function executes successfully
  - `{:aborted, Reason}` if the function failed
 
+In particular, if the table already exists, the reason will be of the form `{:already_exists, table}` so if we try to create this table a second time, we will get:
+
+```shell
```

Comment:
Yeah, I suppose @ybur-yug is right, `elixir` should be used here instead of `shell`. IIRC there were several PRs replacing `shell` with `elixir` before `iex>` in order to enable the syntax highlighting and removing the `iex` line numbers.